sql/mob_pools.sql

```diff
@@ -5954,7 +5954,7 @@ INSERT INTO `mob_pools` VALUES (5895,'Donggu','Donggu',116,0x0000780100000000000
 -- Trusts
 INSERT INTO `mob_pools` VALUES (5896,'shantotto','Shantotto',153,0x0000B80B00000000000000000000000000000000,4,0,2,240,40,1,0,0,0,0,0,32,0,3,0,0,308,0,0,1011,153);
 INSERT INTO `mob_pools` VALUES (5897,'naji','Naji',149,0x0000B90B00000000000000000000000000000000,1,0,2,240,100,0,0,0,0,0,0,32,0,3,0,0,309,0,0,1012,149);
-INSERT INTO `mob_pools` VALUES (5898,'kupipi','Kupipi',153,0x0000BA0B00000000000000000000000000000000,3,0,11,230,40,1,0,0,0,0,0,32,0,3,0,0,310,0,0,1013,153);
+INSERT INTO `mob_pools` VALUES (5898,'kupipi','Kupipi',153,0x0000BA0B00000000000000000000000000000000,3,0,11,230,40,3,0,0,0,0,0,32,0,3,0,0,310,0,0,1013,153);
```

Comment:
Kupipi's behavior is somewhat dependent on where in the order she was summoned, a sign that we need to have some conditional logic we can control rather than static setting this in a field we already used for something else (I was just discussing with Zach how we should not have 1 sql field with 2 enums)

Reply:
I'm not sure how far back "mid" range is vs long, but I also know Kupipi doesn't stand as far back as Apururu even when summoned dead last.

Reply:
I'd leave her be till we sort it out. Since it wasn't 0, someone set her to 1 on purpose.

Reply:
![reminder](https://github.com/LandSandBoat/server/assets/6871475/aa7fe253-e415-45e3-b533-aa7e9f67d68b)

Reply:
its fairly new